aws/validators_test.go

```diff
@@ -2,11 +2,11 @@ package aws
 
 import (
 	"fmt"
+	"github.com/aws/aws-sdk-go/service/cognitoidentity"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
 	"regexp"
 	"strings"
 	"testing"
-
-	"github.com/aws/aws-sdk-go/service/cognitoidentity"
```

Comment:
Nit: The old style of [separating third party libraries is preferred in Go](https://github.com/golang/go/wiki/CodeReviewComments#imports).